### PR TITLE
Localization implementation #500 2nd try - CurrentIncome.js and others

### PR DESCRIPTION
**Pull request by ethanstrominger**
Original pull request was https://github.com/codeforboston/cliff-effects/pull/627.  This somehow got messed up, so created this new pull request.  To review previous comments, see https://github.com/codeforboston/cliff-effects/pull/627.   I purposefully did not change Prev Next as knod said that would be changed to a shared element.

### CODE REVIEW

**Comment by ethanstrominger** [comment]
I purposefully did not change Prev and Next, as knod said that would change to be shared across screens.

**Comment by ethanbb** [comment]
Wait, what happened since I reviewed? Why is it also showing the changes from #629 now?

**Comment by knod** [comment]
@ethanbb : The changes from that PR were accepted to 'dev' and @ethanstrominger pulled them in. Is that what you mean?

**Comment by knod** [comment]
I'm not sure if it runs the tests on the merged versions or not.

**Comment by ethanbb** [comment]
@knod I mean it's showing those changes in the diff now along with Ethan Strominger's changes.

**Comment by knod** [comment]
@ethanbb : That is pretty weird. Those changes should have already taken effect. Hmm.

**Comment by knod** [comment]
My own practice-run didn't create the same diff. This is weird.

File: src/localization/en.js [hunk 1]
```diff
@@ -153,6 +153,9 @@ export default {
     },
 
     currentIncome: {
+      title:        'Current Household Income',
+      clarifier:    'Income that you collected in the past 12 months.',
```
Comment (knod) [inline src/localization/en.js]: This and the line above are incorrectly indented and will create a merge conflict unless fixed before the next PR. You can probably just run `npm run lint`.

**Comment by knod** [comment]
This one's being subsumed by #650. Thanks for this!

**Pull request closed without merging**